bin/trigger_changelog.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2022 Shadow Robot Company Ltd - All Rights Reserved.
+# Proprietary and Confidential. Unauthorized copying of the content in this file, via any medium is strictly prohibited.
+import json
+import argparse
+import requests
+import re
+import time
+
+
+class Constants:
+    URL_BEGINNING = "https://raw.githubusercontent.com"
+    REPOSITORY_ROSINSTALL = "repository.rosinstall"
+    REGEXP_IMAGE_NAME = "image="
+    REGEXP_BRANCH_STORY = "SRC-[^_]+(?=_)"
+    REGEXP_OWNER_REPO = "github.com\/(.*)\/(.*)\.git"
+    REGEXP_TITLE_CONTINUED = "^(.*?)\\r"
+    GITHUB_API_URL = "https://api.github.com"
+    GIT_USERNAME = ""
+    GIT_TOKEN = ""
+    EXCLUDED_REPOS = ["sr_hand_config"]
```

Comment:
Will PR's towards documentation repositories also be listed for the changelog? I was wondering if we want this as with every triggered build the documentation will be updated anyways (even because of the changelog update).

Reply:
It will, it could be worth removing the documentation repos as well as hand_config